packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -106,10 +115,18 @@ export default function authMiddlewareBase(
     pendingTasks,
     requestState,
     tokenCache,
+    fetch: fetcher,
   }: AuthMiddlewareBaseOptions,
   next: Next,
   userOptions?: AuthMiddlewareOptions | PasswordAuthMiddlewareOptions
 ) {
+  if (!fetcher && typeof fetch === 'undefined')
```

Comment:
Will this work in node too, if someone included a polyfill? 

Or should this test for `global.fetch` or `window.fetch`?

Reply:
To my knowledge it should. In the browser the default global is `window` in node it's `global`.